azurerm/resource_arm_key_vault_test.go

```diff
@@ -125,7 +125,6 @@ func TestAccAzureRMKeyVault_update(t *testing.T) {
 					testCheckAzureRMKeyVaultExists(resourceName),
 					resource.TestCheckResourceAttr(resourceName, "access_policy.0.key_permissions.0", "create"),
 					resource.TestCheckResourceAttr(resourceName, "access_policy.0.secret_permissions.0", "set"),
-					resource.TestCheckResourceAttr(resourceName, "tags.environment", "Production"),
```

Comment:
can we add this back in but instead check that `tags.%` is equal to `0` which will ensure there's no tags in the Pre phase